earthpy/clip.py

```diff
@@ -77,7 +77,7 @@ def clip_line_poly(shp, clip_obj):
 
     # Clip the data - with these data
     clipped = shp_sub.copy()
-    clipped['geometry'] = shp_sub.intersection(poly)
+    clipped["geometry"] = shp_sub.intersection(poly)
```

Comment:
@betatim should't dataframe cols be single quotes? thought that was the rule! maybe i am wrong.

Reply:
ok i get it. they like double quotes every.where. ok fine. :)